src/ResourceManagement/Logic/Logic.Tests/InMemoryTests/IntegrationAccountSessionInMemoryTests.cs

```diff
@@ -0,0 +1,316 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+namespace Test.Azure.Management.Logic
+{
+    using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using Microsoft.Rest;
+    using Microsoft.Rest.Azure;
+    using Microsoft.Azure.Management.Logic.Models;
+    using Microsoft.Azure.Management.Logic;
+    using Xunit;
+
+    public class IntegrationAccountSessionInMemoryTests : BaseInMemoryTests
```

Comment:
Renaming this existing base class as suggested and further applied the same naming convention to scenario tests base class. Cleaned-up some include in the modified test files while there.